python/example_code/aurora/aurora_wrapper.py

```diff
@@ -0,0 +1,403 @@
+                                       engine versions to those that are compatible with
+                                       this parameter group family.
+        :return: The list of database engine versions.
+        """
+        try:
+            kwargs = {'Engine': engine}
+            if parameter_group_family is not None:
+                kwargs['DBParameterGroupFamily'] = parameter_group_family
+            response = self.rds_client.describe_db_engine_versions(**kwargs)
+            versions = response['DBEngineVersions']
+        except ClientError as err:
+            logger.error(
+                "Couldn't get engine versions for %s. Here's why: %s: %s", engine,
+                err.response['Error']['Code'], err.response['Error']['Message'])
+            raise
+        else:
+            return versions
+    # snippet-end:[python.example_code.aurora.DescribeDBEngineVersions]
+
+    # snippet-start:[python.example_code.aurora.DescribeOrderableDBInstanceOptions]
+    def get_orderable_instances(self, db_engine, db_engine_version):
+        """
+        Gets DB instance options that can be used to create DB instances that are
+        compatible with a set of specifications.
+
+        :param db_engine: The database engine that must be supported by the instance.
+        :param db_engine_version: The engine version that must be supported by the instance.
+        :return: The list of instance options that can be used to create a compatible instance.
+        """
+        try:
+            inst_opts = []
+            paginator = self.rds_client.get_paginator('describe_orderable_db_instance_options')
+            for page in paginator.paginate(Engine=db_engine, EngineVersion=db_engine_version):
+                inst_opts += page['OrderableDBInstanceOptions']
+        except ClientError as err:
+            logger.error(
+                "Couldn't get orderable instances. Here's why: %s: %s",
+                err.response['Error']['Code'], err.response['Error']['Message'])
+            raise
+        else:
+            return inst_opts
+    # snippet-end:[python.example_code.aurora.DescribeOrderableDBInstanceOptions]
+
+    # snippet-start:[python.example_code.aurora.DescribeDBInstances]
+    def get_db_instance(self, instance_id):
+        """
+        Gets data about a DB instance.
+
+        :param instance_id: The ID of the instance to retrieve.
+        :return: The retrieved DB instance.
+        """
+        try:
+            response = self.rds_client.describe_db_instances(
+                DBInstanceIdentifier=instance_id)
+            db_inst = response['DBInstances'][0]
+        except ClientError as err:
+            if err.response['Error']['Code'] == 'DBInstanceNotFound':
+                logger.info("Instance %s does not exist.", instance_id)
+            else:
+                logger.error(
+                    "Couldn't get db instance %s. Here's why: %s: %s", instance_id,
```

Comment:
Here and throughout: 
db instance --> DB instance

Reply:
fixed